src/redismodule.h

```diff
@@ -131,6 +131,7 @@ typedef long long ustime_t;
 
 #define REDISMODULE_CONFIG_MEMORY (1ULL<<7) /* Indicates if this value can be set as a memory value */
 #define REDISMODULE_CONFIG_BITFLAGS (1ULL<<8) /* Indicates if this value can be set as a multiple enum values */
+#define REDISMODULE_CONFIG_UNPREFIXED (1ULL<<9) /* Indicates if this value needed to be prefixed with the module name */
```

Comment:
Is it related to the value or the config?

Reply:
It is explained in the pull request description. Maybe i should clarify the comment by replacing value with config-name?